zerver/lib/actions.py

```diff
@@ -1133,7 +1133,7 @@ def get_service_bot_events(sender: UserProfile, service_bot_tuples: List[Tuple[i
         # PM triggers for personal and huddle messsages
         elif (not is_stream) and (user_profile_id in active_user_ids):
             trigger = 'private_message'
-        else:
+        else:  # nocoverage
```

Comment:
This line is actually covered by the new test added in this commit.

Reply:
There's some kind of strange bug in the coverage report, because I have run the coverage report like 5 times, and it's always pink, yet when I add a print statement there, it's clearly going through `continue`.  (This is without "nocoverage".)  I removed the "nocoverage" for now, but I don't understand what's going on.

Reply:
Hmm, it was definitely covered in the report when I ran just the single test that should cover it with `--coverage` for me.